invenio_swh/schema.py

```diff
@@ -34,4 +38,28 @@ class Meta:
             "codeRepository",
             "programmingLanguage",
             "developmentStatus",
+            "deposit",
         )
+
+    def add_deposit_info(self, obj):
+        """Add deposit information to the codemeta."""
+        if hasattr(obj, "parent"):
+            parent_record = obj.parent
+        else:
+            parent_record = obj["parent"]
+
+        parent_doi = parent_record.get("pids", {}).get("doi")["identifier"]
+        prefix = current_app.config["DATACITE_PREFIX"]
+        doi_format = current_app.config["DATACITE_FORMAT"]
+        parent_doi = doi_format.format(prefix=prefix, id=parent_doi)
+        origin_obj = {"swh:origin": {"@url": f"https://doi.org/{parent_doi}"}}
```

Comment:
Software Heritage is rejecting this until they update the configuration on their side. 

The current configuration is set to accept depositions with origin `https://doi.org/zenodo/` (note that trailing slash)